package.json

```diff
@@ -1,8 +1,8 @@
 {
-  "name": "lambda-video-thumbnailer",
-  "version": "2.0.0-alpha",
+  "name": "lambda-video-thumbnailer2",
```

Comment:
i dont know why you are changing the version and package name

Reply:
I needed changes in the published package, and there was no response to the pull request for a long time.